quickstart/gcp/setup.md

```diff
@@ -56,4 +31,10 @@ when creating a resource
 creating a resource.
 
 If these values are not provided, some resources may require you to provide these
-values as `project`, `region`, or `zone` properties.
+values as `project`, `region`, or `zone` properties.  You may also set these values per project using `pulumi config`:
+
+```bash
+$ pulumi config set gcp:project <your-project-here>
+$ pulumi config set gcp:region <your-region-here>
+$ pulumi config set gcp:region <your-zone-here>
```

Comment:
Did you mean to say `pulumi config set gcp:zone <your-zone-here>`, instead of `... gcp:region ...`?